scripts/runironic-api

```diff
@@ -4,4 +4,6 @@ export IRONIC_DEPLOYMENT="API"
 
 . /bin/configure-ironic.sh
 
-exec /usr/bin/ironic-api --config-file /usr/share/ironic/ironic-dist.conf ${IRONIC_CONFIG_OPTIONS}
+python3 -c 'import os; import sys; import jinja2; sys.stdout.write(jinja2.Template(sys.stdin.read()).render(env=os.environ))' < /etc/httpd-ironic-api.conf.j2 > /etc/httpd/conf.d/ironic.conf
+
+. /bin/runhttpd
```

Comment:
I have the terrible feeling this breaks installer
looks like a port conflict
(98)Address already in use: AH00072: make_sock: could not bind to address [::]:80

Reply:
Yes, this will break. But if we include this here, we have to discontinue using the httpd container both in installer and CBO (for day 2).